src/field_theory/galois.lean

```diff
@@ -44,19 +44,16 @@ variables (F : Type*) [field F] (E : Type*) [field E] [algebra F E]
 
 /-- A field extension E/F is galois if it is both separable and normal -/
 class is_galois : Prop :=
-(separable' : is_separable F E)
-(normal' : normal F E)
+(to_is_separable : is_separable F E)
+(to_normal : normal F E)
```

Comment:
```suggestion
[to_is_separable : is_separable F E]
[to_normal : normal F E]
```
Does that make sense?

Reply:
I'm not convinced. That makes it harder to write nontrivial instances for `is_galois` such as `is_galois.of_alg_equiv`, since the `[]` brackets mean the arguments to the constructor are inferred as instances. So you'd have to write `@is_galois.mk _ _ _ _ _ sep norm` instead of `⟨sep, norm⟩`.

Reply:
(And the projections don't become instances automatically.)

Reply:
But otoh, you can write instances like `{ }` and TC will take care of the rest. Or you write
```lean
{ to_is_separable := _,
  to_normal := _ }
```
if TC isn't taking care of the rest.

Anyway, I don't care too much. So I'll leave it up to you.

Reply:
Eh, I don't care too much either, so I went ahead and changed it.